index.html

```diff
@@ -105,7 +105,7 @@ <h1 class="title has-text-white">We have web accessibility in
         </ul>
       </nav>
 
-      <section class="section" id="subscription">
+      <section class="section" id="subscription" aria-label="Subscription section">
```

Comment:
I decided to use `aria-label` instead of `aria-labelledby` for `h2` title, because I want to specify that it's section and screen reader will not read title content 2 times.

Reply:
But the screenreader call all sections as region, so maybe the word section is not needed :)

Reply:
Yeah, my point was to add context for screen reader that is a section, but as screen reader call it `region` and it's already clear. So it's better to use `aria-labelledby` for such cases to avoid title duplication.